File: VB/LoginForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace MultiUserOutlookSync
	Partial Public Class LoginForm
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub LoginForm_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			' TODO: This line of code loads data into the 'dataSet1.Users' table. You can move, or remove it, as needed.
			Me.usersTableAdapter.Fill(Me.dataSet1.Users)

		End Sub

		Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			Dim frm As New MainForm(Convert.ToInt32(comboBox1.SelectedValue), comboBox1.Text)
			frm.ShowDialog()
		End Sub
	End Class
End Namespace

File: CS/LoginForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace MultiUserOutlookSync
{
    public partial class LoginForm : Form
    {
        public LoginForm()
        {
            InitializeComponent();
        }

        private void LoginForm_Load(object sender, EventArgs e)
        {
            // TODO: This line of code loads data into the 'dataSet1.Users' table. You can move, or remove it, as needed.
            this.usersTableAdapter.Fill(this.dataSet1.Users);

        }

        private void simpleButton1_Click(object sender, EventArgs e)
        {
			MainForm frm = new MainForm(Convert.ToInt32(comboBox1.SelectedValue), comboBox1.Text);
			frm.ShowDialog();
        }
    }
}